src/styles/variables.js

```diff
@@ -0,0 +1,7 @@
+// 혜린: 제가 전역으로 사용할 mixin들을 여기에 생성해놨습니다. conflict 발생하면 참고해주세요!
+export const flex = obj => `
+	  dispaly : flex;
+    flex-direction : ${obj?.direction ?? "column"};
+	  justify-content : ${obj?.justify ?? "center"}; 
+	  align-items : ${obj?.align ?? "center"}; 
+ 	 `;
```

Comment:
```suggestion
export const flex = ({direction,justify,align}) => `
	  dispaly : flex;
    flex-direction : ${direction ?? "column"};
	  justify-content : ${justify ?? "center"}; 
	  align-items : ${align ?? "center"}; 
 	 `;
```

?? 이 연산자가 어떤 역할을 하는지 reply로 적어주세요~

Reply:
옵셔널 체이닝으로, direction, justify, align이 없으면 찾지 말고 Undefined를 나타내라는 뜻으로 알고 있습니당,,